DefaultSettings.php

```diff
@@ -12,28 +12,37 @@
 	die( 'This file is part of the SemanticExtraSpecialProperties extension, it is not a valid entry point.' );
 }
 
-/**
- * MessageCache
- */
-$GLOBALS['sespCacheType'] = CACHE_ANYTHING;
+return array(
 
-/**
- * To setup properties as fixed properties in order to improve data access
- */
-$GLOBALS['sespUseAsFixedTables'] = false;
+	/**
```

Comment:
@mwjames Since this is a 2.0 refactoring I suggest using the opportunity to enforce a consistent configuration parameter naming. Currently we have three different naming approaches none of them using the scheme used for most other semantic extensions, i.e. `sespg...`. I believe that I am talking to someone thinking alike in this respect. :)